docs/tutorials/10_lattice_models.ipynb

```diff
@@ -894,50 +902,99 @@
+    "    uniform_onsite_potential=v,\n",
+    "    onsite_interaction=u,\n",
+    ")\n",
+    "\n",
+    "lmp = LatticeModelProblem(lattice_model=fhm)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 24,
+   "metadata": {},
    "outputs": [
     {
-     "data": {
-      "text/html": [
-       "<h3>Version Information</h3><table><tr><th>Qiskit Software</th><th>Version</th></tr><tr><td><code>qiskit-terra</code></td><td>0.19.0.dev0+fb934b7</td></tr><tr><td><code>qiskit-aer</code></td><td>0.9.0</td></tr><tr><td><code>qiskit-ignis</code></td><td>0.7.0</td></tr><tr><td><code>qiskit-nature</code></td><td>0.3.0</td></tr><tr><th>System information</th></tr><tr><td>Python version</td><td>3.8.12</td></tr><tr><td>Python compiler</td><td>Clang 13.0.0 (clang-1300.0.29.3)</td></tr><tr><td>Python build</td><td>default, Nov 12 2021 16:34:26</td></tr><tr><td>OS</td><td>Darwin</td></tr><tr><td>CPUs</td><td>4</td></tr><tr><td>Memory (Gb)</td><td>16.0</td></tr><tr><td colspan='2'>Wed Dec 08 19:13:53 2021 JST</td></tr></table>"
-      ],
-      "text/plain": [
-       "<IPython.core.display.HTML object>"
-      ]
-     },
-     "metadata": {},
-     "output_type": "display_data"
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "=== GROUND STATE ===\n",
+      " \n",
+      "* Lattice ground state energy : -2.566350190841\n"
+     ]
     },
     {
-     "data": {
-      "text/html": [
-       "<div style='width: 100%; background-color:#d5d9e0;padding-left: 10px; padding-bottom: 10px; padding-right: 10px; padding-top: 5px'><h3>This code is a part of Qiskit</h3><p>&copy; Copyright IBM 2017, 2021.</p><p>This code is licensed under the Apache License, Version 2.0. You may<br>obtain a copy of this license in the LICENSE.txt file in the root directory<br> of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.<p>Any modifications or derivative works of this code must retain this<br>copyright notice, and modified files need to carry a notice indicating<br>that they have been altered from the originals.</p></div>"
-      ],
-      "text/plain": [
-       "<IPython.core.display.HTML object>"
-      ]
-     },
-     "metadata": {},
-     "output_type": "display_data"
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "/Users/atsushi/github/nature/qiskit_nature/problems/second_quantization/lattice/models/fermi_hubbard_model.py:123: UserWarning: The default value for `display_format` will be changed from 'dense' to 'sparse' in version 0.3.0. Once that happens, you must specify display_format='dense' directly.\n",
+      "  return FermionicOp(ham, register_length=register_length, display_format=display_format)\n"
+     ]
     }
    ],
+   "source": [
+    "from qiskit.algorithms import NumPyMinimumEigensolver\n",
+    "from qiskit_nature.algorithms import GroundStateEigensolver\n",
+    "from qiskit_nature.mappers.second_quantization import JordanWignerMapper\n",
```

Comment:
If I recall correctly, the type of lattice dictates what mapper to use, correctly? Is this being discussed anywhere in this tutorial? If not, this could be an addition for the future (but there is no direct need to add this now as part of this PR)

Reply:
hmm, I have not mentioned about it in this tutorial (I didn't even know that). I will leave it for the future.